src/app/config/url.config.json

```diff
@@ -16,9 +16,9 @@
         "entityConfigUrl" : "/project/v1/configurations/read",
         "spotlightUrl":"/project/v1/template/list?hasSpotlight=true",
         "myImprovementsUrl":"/project/v1/userProjects/list?submitted=false",
-        "recommendationUrl":"/project/v1/library/categories/projects/educationLeader",
+        "recommendationUrl":"",
         "filterUrl": "/project/v1/projectAttributes/find",
-        "savedListUrl": "/project/v1/wishlist/list",
+        "savedListUrl": "/project/v1/wishlist/list?language=en",
```

Comment:
do we have to hardcode this language ?

Reply:
in the next sprint, this will be removed as language will be sent dynamically